intelmq/bots/experts/modify/examples/morefeeds.conf

```diff
@@ -226,6 +226,15 @@
         }
     },
     {
+        "rulename": "ramnit",
+        "if": {
+            "malware.name": "^ramnit-0x.*$"
```

Comment:
> So a more strict rule would be "^ramnit-0x[0-9a-f]{8}$" but I don't know if this is really necessary and it will probably fail to catch malware names if the length of the hex value changes.

So what about `"^ramnit-0x[0-9a-f]*$"`? This would not match malfomed values like `"ramnit-0x00 or something else"` or `"ramnit-0x123;foobar"`